### ISSUE TITLE
Definir escopo técnio primordial

### ISSUE THREAD

**Issue by GabrielFRails** [GabrielFRails/TFD-POO#1]
Qual banco de dados usar?
...

**Comment by Vivianyribeiro** [comment]
Minha sugestão... Postgresql.

**Comment by GabrielFRails** [comment]
> 

Pode pesquisar pra gente como usar ele com python?

**Comment by Vivianyribeiro** [comment]
Sim, mas se vc ja usa algum com python ja, podemos usar oq tu tem mais afinidade.

**Comment by Vivianyribeiro** [comment]
Só uma duvida, a gente vai baixar algum banco com dados de mercadorias e tal ou vamos criar nosso próprio

**Comment by GabrielFRails** [comment]
Vamos criar o nosso próprio, seria outra coisa a se discutir futuramente, o modelo de dados do nosso CRUD simples e depois aplicá-lo aos endpoints.

Creio que não precisa ser uma base de dados grande, apenas um tanto que seria o suficiente basta

**Comment by Vivianyribeiro** [comment]
Instalar a biblioteca - pip install psycopg2
import psycopg2  - > Esse pacote, faz a comunicação do Python com banco de dados PostgreSQL. Dessa forma é possível realizar operações no banco de dados via scripts Python. Estas funcionalidades facilitam e muito a manipulação de dados na linguagem Python e os bancos de dados PostgreSQL.
Peguei essas info no site : https://dadosaocubo.com/manipulando-dados-em-postgresql-com-python/
